dashboard/app/models/programming_expression.rb

```diff
@@ -305,10 +305,10 @@ def get_localized_params
     i18n_params = I18n.t(
       :palette_params,
       scope: expression_scope,
-      default: palette_params,
+      default: nil,
       smart: true
     )
-    if i18n_params != localized_params
+    unless i18n_params.nil?
```

Comment:
Exactly right.

Reply:
An alternative would be to have `default: {}` and then remove the condition altogether.

Reply:
I'd rather not process the content if translations aren't available. We would only iterate over the parameters when there's a translation, avoiding errors such as the one found in honeybadger.